doc/tools.md

```diff
@@ -92,6 +92,18 @@ mySetting: test value
 
 With the first file, you'd start the server with `zat server -c settings.json`. With the second file, you'd start it with `zat server -c`.
 
+To access the settings from your JavaScript code, use the `setting('name')` framework method to retrieve the value of a setting. Example:
+
+```javascript
+alert( this.setting('mySetting') );
+```
+
+To access the settings from your Handlebars templates, use the `{{setting 'name'}}` [template helper](./templates#setting-name). Example:
```

Comment:
@chucknado - I copied this link from another help page - but it is not clear to me how the doc pages (located in different git depots) can have relative references to one another - Is this correct?